code/datums/gamemodes/distress.dm

```diff
@@ -456,49 +440,62 @@
-						/obj/item/storage/pouch/construction = round(scale * 2),
-						/obj/item/storage/pouch/tools = round(scale * 2),
-						/obj/item/storage/pouch/explosive = round(scale * 2),
-						/obj/item/storage/pouch/syringe = round(scale * 2),
-						/obj/item/storage/pouch/medical = round(scale * 2),
-						/obj/item/storage/pouch/medkit = round(scale * 2),
+						/obj/item/clothing/tie/storage/webbing = round(scale * 5),
+						/obj/item/clothing/tie/storage/brown_vest = round(scale * 5),
+						/obj/item/clothing/tie/storage/white_vest/medic = round(scale * 5),
+						/obj/item/clothing/tie/holster = round(scale * 5),
+						/obj/item/storage/pouch/general/medium = round(scale * 5),
+						/obj/item/storage/pouch/general/large = round(scale * 2),
+						/obj/item/storage/pouch/construction = round(scale * 5),
+						/obj/item/storage/pouch/tools = round(scale * 5),
+						/obj/item/storage/pouch/explosive = round(scale * 5),
+						/obj/item/storage/pouch/syringe = round(scale * 5),
+						/obj/item/storage/pouch/medical = round(scale * 5),
+						/obj/item/storage/pouch/medkit = round(scale * 5),
 						/obj/item/storage/pouch/magazine = round(scale * 5),
+						/obj/item/storage/pouch/magazine/large = round(scale * 2),
 						/obj/item/storage/pouch/flare/full = round(scale * 5),
 						/obj/item/storage/pouch/firstaid/full = round(scale * 5),
 						/obj/item/storage/pouch/pistol = round(scale * 10),
+						/obj/item/storage/pouch/magazine/pistol = round(scale * 10),
 						/obj/item/storage/pouch/magazine/pistol/large = round(scale * 5),
 						/obj/item/storage/pouch/shotgun = round(scale * 10),
 						/obj/item/weapon/gun/pistol/m4a3 = round(scale * 20),
-						/obj/item/weapon/gun/pistol/m1911 = round(scale * 2),
+						/obj/item/weapon/gun/pistol/m1911 = round(scale * 5),
 						/obj/item/weapon/gun/revolver/m44 = round(scale * 10),
 						/obj/item/weapon/gun/smg/m39 = round(scale * 15),
 						/obj/item/weapon/gun/rifle/m41a = round(scale * 20),
 						/obj/item/weapon/gun/shotgun/pump = round(scale * 10),
 						/obj/item/weapon/gun/energy/lasgun/M43 = round(scale * 10),
 						/obj/item/explosive/mine = round(scale * 2),
-						/obj/item/storage/box/nade_box = round(scale * 2),
-						/obj/item/storage/box/nade_box/impact = round(scale * 2),
 						/obj/item/explosive/grenade/frag/m15 = round(scale * 2),
 						/obj/item/explosive/grenade/incendiary = round(scale * 4),
 						/obj/item/explosive/grenade/smokebomb = round(scale * 5),
-						/obj/item/explosive/grenade/cloakbomb = round(scale * 3),
+						/obj/item/explosive/grenade/cloakbomb = round(scale * 4),
+						/obj/item/storage/box/nade_box = round(scale * 2),
 						/obj/item/storage/box/m94 = round(scale * 30),
 						/obj/item/flashlight/combat = round(scale * 5),
 						/obj/item/clothing/mask/gas = round(scale * 10)
 						)
 
 		CG.contraband = list(
-						/obj/item/weapon/gun/smg/ppsh = round(scale * 4),
+						/obj/item/storage/box/nade_box/impact = round(scale * 1),
```

Comment:
Remove these completely. Only make them orderable.
Consider that contraband will always be done if there's interesting things to get there.
We restrict them for now to meme ineffective stuff, like the PPSH.